### ISSUE TITLE
[a11y][FxA] When user enters username/pass and needs email validation, page contents are not announced

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2932]
## Steps to reproduce
- enable voiceview
- Start FxA sign in, entering email and password on an account that requires email validation

### Expected behavior
When email validation is required, that is announced to the user

### Actual behavior
Nothing is announced to the user: **they'll have no idea what to do next.**

This site may be more navigable in review mode (I haven't tested yet) but they may not know to enter it. We could choose to auto enter it for them: #300.

### Device information
* Fire TV device: ?
* Latest affected Firefox version: v4.5+ master 2cc87063e1808eaaa291b818cb22c952c9469607
* Earliest affected Firefox version: ?
* Turbo Mode: ?
